test/metabase/sync/sync_metadata/fields/fetch_metadata_test.clj

```diff
@@ -39,8 +39,8 @@
              {:name              "ts"
               :database-type     "BIGINT"
               :base-type         :type/BigInteger
-              :effective-type    :type/BigInteger
-              :semantic-type     :type/UNIXTimestampMilliseconds
+              :effective-type    :type/DateTime
```

Comment:
I think these should technically come back with effective type `:type/Instant` now but it probably makes zero difference here